install.ps1

```diff
@@ -600,7 +600,7 @@ function Test-AirgapTarballChecksum() {
         return
     }
 
-    if ($ExpectedImageAirGapChecksum) {
+    if (-Not $ExpectedImageAirGapChecksum) {
```

Comment:
The install script in master is working based on my testing https://github.com/rancher/rke2/issues/2543#issuecomment-1062217804, have you verified that you are running powershell as an administrator?

Reply:
I had an incorrect understanding of the issue, removing my block and approving as I have validated this fix myself as well.